examples/ssx-quickstart-express/src/index.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import http from 'http';
+import { AddressInfo } from 'net';
+import { SSXServer, SSXExpressMiddleware } from '@spruceid/ssx-server';
+import cors from 'cors';
+
+const app = express();
+
+const ssx = new SSXServer({
+  signingKey: "SUPER SECRET",
+});
+
+app.use(cors({ credentials: true, origin: true }));
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(SSXExpressMiddleware(ssx));
+
+app.get('/', (_req, res) => {
+  res.send('UP\n');
+});
+
+const server = http.createServer(app).listen({ host: '0.0.0.0', port: '5000' }, () => {
```

Comment:
Any reason to not do something like this and have fewer dependencies?
```js
app.listen(port, () => {
  console.log(`⚡️[server]: Server is running at http://localhost:${port}`);
});
```

Reply:
actually no, will make the change. I was using the object earlier but it lost the purpose